lists/gb/gb-chc.json

```diff
@@ -24,32 +24,44 @@
   "listType": "primary",
   "access": {
     "availableOnline": true,
-    "onlineAccessDetails": "",
-    "publicDatabase": "https://www.gov.uk/government/organisations/charity-commission",
-    "guidanceOnLocatingIds": "",
+    "onlineAccessDetails": "Information about the data and information available through the search is available on the page listed as the list URL. Click through to \"Find a charity\" for a simple text search of a charity's name. Use the advanced search to filter the search across multiple different aspects of the dataset.",

```

Comment:
Suggestion: replace '...information available through the search is available on the page listed as the list URL' with 

'...information available through the search is available at https://register-of-charities.charitycommission.gov.uk'.

Reply:
Thanks @kd-ods those suggestions make the text a lot clearer. I've made the changes as recommended.